doit/apps/tasks/tests/v1_tests.py

```diff
@@ -52,3 +52,9 @@ def test_retrieve_task(self):
             response.data.get('description', None),
             self.dummy_data.get('description', None)
         )
+
+    def test_delete_task(self):
+        url = reverse('v1:tasks-detail', kwargs={'pk': self.task.id})
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
Response hatalı: https://github.com/crownest/doit-backend/blob/staging/docs/api/v1.md#task-delete

Bu şekilde test'in patlaması lazım.